All About C/Intro to OOPS/6. this Pointer.cpp

```diff
@@ -25,9 +26,10 @@ this-> width=width;
 
 int main(){
 
-rectangle a(5,6);
-
-cout<<a.area()<<endl;
+rectangle a(5,6);// rectangle class is called 
```

Comment:
Error: Kindly provide proper code

Reply:
This was my first contri so i added comments should i properly comment is whole code ?

Reply:
you need to write class for rectangle, so that it wouldn't create error👍🏻